metacat-connector-hive/src/test/groovy/com/netflix/metacat/connector/hive/HiveConnectorDatabaseSpec.groovy

```diff
@@ -41,160 +40,174 @@ import spock.lang.Unroll
+        def hiveConnectorDatabaseService = new HiveConnectorDatabaseService("testhive", client, new HiveConnectorInfoConverter(new HiveTypeConverter()))
         when:
-        def result = hiveConnectorDatabaseService.exists(connectorContext, QualifiedName.ofDatabase("testhive","testdb"))
+        def result = hiveConnectorDatabaseService.exists(connectorContext, QualifiedName.ofDatabase("testhive", "testdb"))
         then:
-        client.getDatabase(_) >> {throw new NoSuchObjectException()}
+        client.getDatabase(_) >> { throw new NoSuchObjectException() }
         result == false
     }
 
-    def "Test for exist database TException"(){
+    def "Test for exist database TException"() {
         def client = Mock(MetacatHiveClient)
-        def hiveConnectorDatabaseService = new HiveConnectorDatabaseService("testhive", client, new HiveConnectorInfoConverter( new HiveTypeConverter() ) )
+        def hiveConnectorDatabaseService = new HiveConnectorDatabaseService("testhive", client, new HiveConnectorInfoConverter(new HiveTypeConverter()))
         when:
-        def result = hiveConnectorDatabaseService.exists(connectorContext, QualifiedName.ofDatabase("testhive","testdb"))
+        def result = hiveConnectorDatabaseService.exists(connectorContext, QualifiedName.ofDatabase("testhive", "testdb"))
         then:
-        client.getDatabase(_) >> {throw new TException() }
+        client.getDatabase(_) >> { throw new TException() }
         thrown ConnectorException
     }
 
     @Unroll
-    def "Test for listNames database with Exceptions"(){
+    def "Test for listNames database with Exceptions"() {
         def client = Mock(MetacatHiveClient)
-        def hiveConnectorDatabaseService = new HiveConnectorDatabaseService("testhive", client, new HiveConnectorInfoConverter( new HiveTypeConverter() ) )
+        def hiveConnectorDatabaseService = new HiveConnectorDatabaseService("testhive", client, new HiveConnectorInfoConverter(new HiveTypeConverter()))
 
         when:
-        def dbs = hiveConnectorDatabaseService.listNames(connectorContext, QualifiedName.ofDatabase("testhive","testdb"), null, new Sort(null, SortOrder.ASC), null )
+        def dbs = hiveConnectorDatabaseService.listNames(connectorContext, QualifiedName.ofDatabase("testhive", "testdb"), null, new Sort(null, SortOrder.ASC), null)
         then:
-        1 * client.getAllDatabases() >> {throw exception}
+        1 * client.getAllDatabases() >> { throw exception }
         thrown result
 
         where:
-        exception                    | result
-        new TException()             |ConnectorException
-        new MetaException()          |InvalidMetaException
+        exception           | result
+        new TException()    | ConnectorException
+        new MetaException() | InvalidMetaException
 
     }
 
-    @Ignore
-    def "Test for get database" (){
+    def "Test for get database not found"() {
```

Comment:
@ajoymajumdar @zhljen Not this test and the next "Test for get database" the previous behavior didn't seem to make sense to me. Basically the MetacatHiveClient would return a null for the get database and we'd just return a database info object that regurgitated the QualifiedName from the info converter even though I would assume a null from the MetacatHiveClient should mean the database doesn't actually exist. Hence when I re-enabled this test it kept throwing NullPointerException after I re-enabled it cause I'd added `@NonNull` to the database parameter. If this was some sort of expected behavior we can change it back but like I said it didn't seem to make sense to me.